app/viewmodels/DutyDefermentAccount.scala

```diff
@@ -16,14 +16,23 @@
 
 package viewmodels
 
-case class DutyDefermentAccount(accountNumber: String, statementsForAllEoris: Seq[DutyDefermentStatementsForEori],
+import java.time.LocalDate
+
+case class DutyDefermentAccount(accountNumber: String,
+                                statementsForAllEoris: Seq[DutyDefermentStatementsForEori],
                                 linkId: String, isNiAccount: Boolean) {
 
   val hasRequestedStatements: Boolean = statementsForAllEoris.exists(_.requestedStatements.nonEmpty)
   val hasCurrentStatements: Boolean = statementsForAllEoris.exists(_.currentStatements.nonEmpty)
 
   def firstPopulatedStatement: Option[DutyDefermentStatementsForEori] = statementsForAllEoris.find(_.groups.nonEmpty)
 
+  private val amtMonthsHistory: Int = 6
```

Comment:
Please move this val on line 31 to line 27 in order to keep all the vals together